examples/tutorials/vectors.py

```diff
@@ -0,0 +1,486 @@
+
+fig.plot(
+    data=data,
+    style="m0.5c+ea",
+    pen="2p",
+    color="red3",
+)
+fig.show()
+
+########################################################################################
+# When plotting multiple vectors there is a multitude of numpy functions
+# that can make this process easier. In the following example, three main numpy
+# functions are used. The first of which is ``np.arange`` which iterates from 0
+# to the value of ct. This function is used to generate random data points for
+# ``radius`` and ``stopdir`` since these values are intended to be different in
+# each of the five vectors begin plotted. The second function is ``np.full``
+# which creates a ``numpy.ndarray``. Finally, all of this data is congregated into
+# a final ``numpy.ndarray`` called data using ``np.column_stack``. This is then
+# passed to the plot function and the resulting figure is shown below.
+
+ct = 5
+radius = 3 - (0.5 * np.arange(0, ct))
+startdir = np.full(ct, 90)
+stopdir = 180 + (50 * np.arange(0, ct))
+
+data = np.column_stack([np.full(ct, 0), np.full(ct, 0), radius, startdir, stopdir])
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[-5, 5, -5, 5],
+    projection="X10c",
+    frame="ag",
+    data=data,
+    style="m0.5c+ea",
+    pen="2p",
+    color="red3",
+)
+fig.show()
+
+########################################################################################
+# Much like when plotting regular vectors, the default unit used is centimeters.
+# When this is changed to inches, the size of the plot appears larger when the
+# projection units do not change. Below is an example of two circular vectors.
+# One is plotted using the default unit, and the second is plotted using inches.
+# The difference in size of the two vectors provides good insight into how this
+# functionality works.
+
+circular_vector_1 = [6, 5, 2, 90, 270]
+circular_vector_2 = [6, 5, 1, 90, 270]
+
+data_1 = [circular_vector_1]
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X10c",
+    frame="ag",
+    data=data_1,
+    style="m0.5c+ea",
+    pen="2p",
+    color="red3",
+)
+
+data_2 = [circular_vector_2]
+with pygmt.config(PROJ_LENGTH_UNIT="i"):
+    fig.plot(
+        data=data_2,
+        style="m0.5c+ea",
+        pen="2p",
+        color="red3",
+    )
+fig.show()
+
+########################################################################################
+# Plot Geographic Vectors
+# -----------------------
+# Geographic graph using x and y values to set a start and an ending point.
+# Use ``fig.coast`` to plot a coast. ``x`` and ``y`` are coordinates
+# on a grid that we are using. ``x`` is Idaho and ``y`` is Chicago in this example.
+# The geographical vector is going from Idaho to Chicago. To style geographic
+# vectors, use ``=`` at the begining of the ``style`` parameter.
+# Other styling features such as arrow head color and line thickness
+# can be passed into
+# ``pen`` and ``color``parameters
```

Comment:
Vector can have other kinds of heads instead of only arrow.
```suggestion
# Other styling features such as vector stem width and head color
# can be passed into ``pen`` and ``color`` parameters.
```